www/_template/guides/web-worker.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layouts/content.njk
+title: 'Web Workers'
+tags: communityGuide
+published: true
+description: How to use Web Workers in your Snowpack project.
+---
+
+[Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) are a simple means for web content to run scripts in background threads. 
+
+**To use Web Workers with Snowpack:** Use the browser's native [Web Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Web_Workers_API) directly:
+
+```js
+// Example: Load a Web Worker in your project
+const myWorker = new Worker(new URL("./worker.js", import.meta.url));
+```
+
+Passing a `URL` to the `Worker` constructor (instead of a string literal) isn't required to run in the browser. However, Webpack 5 requires this for its automatic bundling and Native Worker support.
```

Comment:
Wait why does Webpack matter here? Is it only if you're using the Webpack plugin?

Reply:
yea, it may be best to just remove this line entirely. It's something to do with how they support automatic web worker bundling, but it actually may be required for ESM module support in web workers as well. 

Or, simplify it to:

> Passing a `URL` to the `Worker` constructor (instead of a string literal) is recommended, but not required. Using a string literal (`new Worker('./worker.js')`) may prevent issues with bundler optimizations when you build for production.